src/model.d.ts

```diff
@@ -1196,6 +1196,11 @@ export interface BulkCreateOptions<TAttributes = any> extends Logging, Transacti
    * Return all columns or only the specified columns for the affected rows (only for postgres)
    */
   returning?: boolean | Array<keyof TAttributes | Literal | Col>;
+
+  /**
+   * Primary key names used as conflict target columns during insertion.
+   */
+  upsertKeys?: string[];
```

Comment:
Looking at `bulkCreate` it seems that it sets this by itself;
https://github.com/sequelize/sequelize/blob/f6045f8f1cc48b8bb01d1ce9df0596083d3ebcf2/src/model.js#L2251-L2253

Can you add a test using this option that shows otherwise?

Reply:
Hi @WikiRik ✋
I'll work on this 💪 

Anyway, I've encountered problems after cloning the repo and running `yarn install`. 
I keep getting this error message:
![Screenshot 2023-02-11 at 14 44 13](https://user-images.githubusercontent.com/15077781/218261275-cf36fc5d-0868-4303-a66c-9ca1f98ca51a.jpg)

I searched online but I didn't find anything helpful.
Do you know how I could solve the problem? Should I open a new issue?

Thanks

Reply:
Can you delete the node_modules folder and try to install it again? If that does not work, you can try opening an issue for the ibm_db project. It's not a sequelize issue